generated/nifake/nifake/unit_tests/test_session.py

```diff
@@ -9,6 +9,7 @@
 import platform
```

Comment:
I think this whole file needs to be split up into `test_session` where the LibraryInterpreter is mocked and `test_library_interpreter`.

Most of the stuff here today would end up in `test_library_interpreter`... all the ctypes handling.

You can open an issue and do it separately so this can be pushed in.

Reply:
AB#2156250